logicrunner/goplugin/ginsider/ginsider.go

```diff
@@ -58,10 +59,21 @@ type RPC struct {
 	GI *GoInsider
 }
 
+func recoverRPC(err *error) {
+	if r := recover(); r != nil {
+		if err != nil && *err == nil {
+			*err = errors.New(fmt.Sprint(r))
```

Comment:
May be is it better to join text of panic and text of error?
Because it looks like we loose text of panic if error exists, but print only text of panic

Reply:
done